### PR TITLE
Bugfixes

### PR DESCRIPTION
**Pull request by jo-asakura**
:eyeglasses: @ajacksified or @danehansen

### CODE REVIEW

File: src/views/components/CommentBox.jsx [hunk 1]
```diff
@@ -4,10 +4,17 @@ import { models } from 'snoode';
...
+      inputCssClass: ''
+    };
   }
 
-  shouldComponentUpdate () {
-    return false;
+  handleInputChange (e) {
+    var textEl = this.refs.text.getDOMNode();
+    this.setState({
+      inputCssClass: textEl.value.trim().length ? 'is-w-content' : ''
```
Comment (ajacksified) [inline src/views/components/CommentBox.jsx]: :no_bell: what does `w-content` stand for?

File: src/routes.jsx [hunk 2]
```diff
@@ -359,12 +352,13 @@ function routes(app) {
 
     var props = buildProps(this, {
       referrer: ctx.headers.referer,
+      title: '400 - Oops, looks like something went wrong.'
```
Comment (ajacksified) [inline src/routes.jsx]: Minor nit: need trailing commas on your properties.

File: src/views/components/CommentBox.jsx (hunk 1 above)
Comment (jo-asakura) [inline src/views/components/CommentBox.jsx]: `with-content` :smile:

**Comment by ajacksified** [comment]
:nail_care:

File: src/views/components/CommentBox.jsx (hunk 1 above)
Comment (ajacksified) [inline src/views/components/CommentBox.jsx]: Maybe rename to either `has-content` or `is-with-content`?

**Comment by jo-asakura** [comment]
:haircut: @ajacksified

File: src/views/components/CommentBox.jsx [hunk 3]
```diff
@@ -4,10 +4,17 @@ import { models } from 'snoode';
 class CommentBox extends React.Component {
   constructor(props) {
     super(props);
+
+    this.state = {
+      inputCssClass: ''
```
Comment (ajacksified) [inline src/views/components/CommentBox.jsx]: comma

**Comment by ajacksified** [comment]
Rebase `cleanup` into your previous commits, then :fish:

**Pull request closed without merging**